src/modals/AddProjectModal.tsx

```diff
@@ -116,8 +111,10 @@ const AddProjectModal = () => {
                   boxShadow: "none",
                   outline: "none"
                 }}
-                value={name}
-                onChange={handleNameChange}
+                value={nameRef.current?.value}
```

Comment:
We don't need an onChange handler on useRef. Watch my video if it is necessary

Reply:
what a beautiful invention this useRef.